netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -289,7 +290,8 @@ public void handlerAdded(final ChannelHandlerContext ctx) throws Exception {
 
     // init max connection age monitor
     if (maxConnectionAgeInNanos != MAX_CONNECTION_AGE_NANOS_DISABLED) {
-      maxConnectionAgeMonitor = ctx.executor().schedule(
```

Comment:
Oh, then make your own answer to call fakeClock directly? (Don't use `delegatesTo()` in that case)